pkg/run/bootstrap/bootstrap_wizard_ui.go

```diff
@@ -242,23 +244,19 @@ func (m wizardModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			// Did the user press enter while the submit button was focused?
 			// If so, exit.
 			if t == tea.KeyEnter && m.focusIndex == len(m.textInputs) {
-				options := []*BootstrapCmdOption{}
+				options := make(BootstrapCmdOptions)
 
 				for i, input := range m.textInputs {
 					prompt := m.prompts[i]
 
 					value := strings.TrimSpace(input.Value())
 
 					if value == "" {
-						continue
+						m.errorMsg = "Missing value in " + input.Placeholder
+						return m, tea.Batch()
```

Comment:
@opudrovs I'm pretty sure I'm violating all the rules here - do you know how I'm supposed to write this?